quipucordsctl.spec

```diff
@@ -1,9 +1,14 @@
+###############################################################
+# Update the following globals for the downstream build:
```

Comment:
Personal style preference: I would remove these comments as they seem redundant because the purpose of the globals seems obvious.